mlflow/tracing/provider.py

```diff
@@ -298,22 +302,22 @@ def _setup_tracer_provider(disabled=False):
     suppress_warning("opentelemetry.sdk.trace", "Calling end() on an ended span")
 
 
-def _get_mlflow_span_processor(tracking_uri: str, experiment_id: Optional[str] = None):
+def _get_mlflow_span_processor(tracking_uri: str, experiment_id: Optional[str] = None, exporter = None):
     """
     Get the MLflow span processor instance that is used by the current tracer provider.
     """
     if is_databricks_uri(tracking_uri):
         from mlflow.tracing.export.mlflow_v3 import MlflowV3SpanExporter
         from mlflow.tracing.processor.mlflow_v3 import MlflowV3SpanProcessor
 
-        exporter = MlflowV3SpanExporter(tracking_uri=tracking_uri)
+        exporter = exporter or MlflowV3SpanExporter(tracking_uri=tracking_uri)
         processor = MlflowV3SpanProcessor(exporter, experiment_id=experiment_id)
```

Comment:
@jamesbxwu @B-Step62 following from https://github.com/mlflow/mlflow/pull/15700/files#r2202190322, when `is_databricks_uri(tracking_uri)` is `True`, this is where we should return `DatabricksMlflowV3SpanExporter`.